tests/unit/conftest.py

```diff
@@ -14,8 +14,10 @@
 
 import pytest
 
+from tests.unit.oauth_test_utils import SERVER_ADDRESS
```

Comment:
Shouldn't we put SERVER_ADDRESS in a more central place if we want to use it in multiple places. here the usage is not related to oauth.